path_handler.py

```diff
@@ -0,0 +1,15 @@
+import os
```

Comment:
Global variables for what? I don't remember having pathing issues with the framework?